.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
```

Comment:
Yes @benlast. To clear up any confusion, the templates under `.github/` are specifically for the code-style repo, while those under `github-templates` are generic ones to apply to the majority of Mobify repos, but can be adapted and modified wherever the dev team sees fit. 

I know we have some disparity between PR templates in various engineering teams at Mobify, so this is a good time to discuss whether or not we want to standardize.